exercises/phone-number/test/Tests.hs

```diff
@@ -23,30 +23,34 @@ data Case = Case { description ::       String
 cases :: [Case]
 cases =
     [ Case { description = "cleans the number"
-           , input       = "(123) 456-7890"
-           , expected    = Just "1234567890"
+           , input       = "(223) 456-7890"
+           , expected    = Just "2234567890"
            }
     , Case { description = "cleans number with dots"
-           , input       = "123.456.7890"
-           , expected    = Just "1234567890"
+           , input       = "223.456.7890"
+           , expected    = Just "2234567890"
            }
     , Case { description = "cleans numbers with multiple spaces"
-           , input       = "123 456   7890   "
-           , expected    = Just "1234567890"
+           , input       = "223 456   7890   "
+           , expected    = Just "2234567890"
            }
     , Case { description = "invalid when 9 digits"
            , input       = "123456789"
            , expected    = Nothing
            }
-    , Case { description = "invalid when 11 digits"
-           , input       = "21234567890"
+    , Case { description = "invalid when 11 digits does not start with a 1"
+           , input       = "22234567890"
            , expected    = Nothing
            }
     , Case { description = "valid when 11 digits and first is 1"
```

Comment:
Since I have a case that says "starting with 1" later on, I might as well change this one to match. Coming right up.